settings.gradle.kts

```diff
@@ -0,0 +1,14 @@
+pluginManagement {
+    repositories {
```

Comment:
Rewrote this file as Kotlin because it helps a bit with build speed, since Groovy build scripts can't be build cached until Gradle 8.7, which is at RC2 (not yet released).